config/e2e.yml

```diff
@@ -0,0 +1,28 @@
+boolTestKey: true
+numberTestKey: 1
+stringTestKey: 'string'
+
+rollbar:
+  accessToken: 'ROLLBAR_ACCESS_TOKEN'
+  env: 'ROLLBAR_ENV'
```

Comment:
This should be removed as we are not using rollbar in e2e

Reply:
Removed it from the config file